pkg/render/fluentd_test.go

```diff
@@ -1051,3 +1086,44 @@ var _ = Describe("Tigera Secure Fluentd rendering tests", func() {
 		)
 	})
 })
+
+func setupEKSConfig() *render.EksCloudwatchLogConfig {
```

Comment:
nit: this should be `setupEKSCloudwatchLogConfig` (if we want to have a helper function at all)